src/components/UserActions/QueryList.ts

```diff
@@ -156,6 +161,18 @@ export class QueryList extends Component {
         }
         return listItem;
     }
+
+    private addTooltipElement(element: HTMLElement, action: UserAction) {
```

Comment:
would it make sense to isolate the logic of this method in a separate file avoid writing  it twice?

Reply:
Yea I thought about it, felt a bit weird to pass in an HTMLElement belonging to this component to another file, and just extracting the logic purely for the tooltip made for very little code to put in another file

Reply:
Also the selector for the insertBeforeElement
`const insertBeforeElement = element.querySelector('.coveo-link');`

changes between the two files.